qutebrowser/mainwindow/tabwidget.py

```diff
@@ -705,7 +705,17 @@ def wheelEvent(self, e):
             e: The QWheelEvent
         """
         if config.val.tabs.mousewheel_switching:
-            super().wheelEvent(e)
+            if utils.is_mac:
+                index = self.currentIndex()
+                if index != -1:
+                    if e.angleDelta().y() > 0:
+                        index += -1
+                    else:
+                        index += 1
+                    if index >= 0 and index < self.count():
```

Comment:
You can chain comparisons in Python to check if something is in a certain range:

```suggestion
                    if 0 <= index < self.count():
```